mcp/tests/playwright-generator.js

```diff
@@ -201,15 +201,32 @@ function generateUserInteractions(
   }
 
   if (assertions && assertions.length > 0) {
+    const ambiguousKeywords = [
+      "iframe",

```

Comment:
why do we have ambiguous keywords from the example iframe hardcoded here?? This is meant to work on any site, not just the example iframe